src/domain/camera.py

```diff
@@ -68,8 +69,8 @@ def detect_motion_from_video(self, test_mode = False):
         width  = int(cap.get(cv2.CAP_PROP_FRAME_WIDTH))
         height = int(cap.get(cv2.CAP_PROP_FRAME_HEIGHT))
         fps    = cap.get(cv2.CAP_PROP_FPS)
-        logger.debug('Length: %.2f | Width: %.2f | Height: %.2f | Fps: %.2f' %
-                      (length, width, height, fps))
+        logger.debug('Length: %.2f | Width: %.2f | Height: %.2f | Fps: %.2f',
```

Comment:
Use " for strings instead of ', to make code homogeneous.
Also below, on all other occurrences.